vagrant/hosts

```diff
@@ -5,6 +5,7 @@
 [all:vars]
 ansible_ssh_user=vagrant
 ansible_dns=mesos.dev
+marathon_port=8080
```

Comment:
Since this is unlikely to be changed I suggest to define it in `roles/mesos/defaults/main.yml`